sway/commands/shadow_color.c

```diff
@@ -17,6 +17,8 @@ struct cmd_results *cmd_shadow_color(int argc, char **argv) {
 		return cmd_results_new(CMD_INVALID, "Invalid %s color %s",
 				"shadow_color", argv[0]);
 	}
+	if (!memcmp(config->shadow_color, config->shadow_inactive_color, sizeof(config->shadow_color)))
+		color_to_rgba(config->shadow_inactive_color, color);
```

Comment:
Why would you set the inactive color here?

Reply:
I did it so if you only set a custom shadow_color, both of them are set to same one (same behaviour as now), but if you do set a shadow_inactive_color then they are separate. If you don’t set it there and only use shadow_color then shadow_inactive_color would still be the default. But I could find another place to do it if that is not fine.

Reply:
I'd prefer if you check if it equals 0 instead of inverting the value. Also add curly brackets after the if-statement

Reply:
Absolutely! Did you mean something like the new commit? Thanks

Reply:
> Absolutely! Did you mean something like the new commit? Thanks

Yeah :)